tests/messages_tests/tests.py

```diff
@@ -185,3 +187,20 @@ def test_mismatching_length(self):
         )
         with self.assertRaisesMessage(AssertionError, msg):
             self.assertMessages(response, [])
+
+    def test_method_frames_ignored_by_unittest(self):
+        response = FakeResponse()
+        add_message(response.wsgi_request, constants.INFO, "INFO message.")
+        try:
+            self.assertMessages(response, [])
+        except AssertionError:
+            exc_type, exc, tb = sys.exc_info()
+
+        result = unittest.TestResult()
+        result.addFailure(self, (exc_type, exc, tb))
+        stack_summary = traceback.extract_tb(exc.__traceback__)
+        self.assertEqual(len(stack_summary), 1)
+        # This method, not assertMessages
+        self.assertEqual(
+            stack_summary[-1].name, "test_method_frames_ignored_by_unittest"
+        )
```

Comment:
Could we just call this `stack` and avoid line wrapping down below? :smile: 
```suggestion
        stack = traceback.extract_tb(exc.__traceback__)
        self.assertEqual(len(stack), 1)
        # Top element in the stack is this method, not assertMessages.
        self.assertEqual(stack[-1].name, "test_method_frames_ignored_by_unittest")
```